cnc_ctrl_v1/Report.cpp

```diff
@@ -237,6 +237,8 @@ void  returnPoz(){
         Serial.print(sys.yPosition/sys.inchesToMMConversion);
         Serial.print(F(","));
         Serial.print(zAxis.read()/sys.inchesToMMConversion);
+        Serial.print(F(","));
+        Serial.print(sys.feedrate/sys.inchesToMMConversion);
```

Comment:
I like that you just pulled this from the existing calculations instead of re-computing it. 👍